doc/api/addons.md

```diff
@@ -708,6 +708,13 @@ console.log(obj.plusOne());
 // Prints: 13
 ```
 
+When it comes to destructing wrapper objects, garbage collector can execute
+forcefully using V8 command line flags ` --gc_global ` and ` --gc_interval `,
+where ` --gc_global ` forces V8 to perform a full garbage collection and
+` --gc_interval ` forces V8 to perform garbage collection after a given amount
+of allocations. Although, it is recommended to limit V8 command line flags
+for testing purposes only, since these are primarily debug flags.
+
```

Comment:
How about this?:

```markdown
The destructor for a wrapper object will run when the object is
garbage-collected. For destructor testing, there are command-line flags that can
be used to force garbage collection. These flags are provided by the underlying
V8 JavaScript engine. They are subject to change or removal at any time. They
are not documented by Node.js or V8, and they should never be used outside of
testing.
```

Or if that's too mysterious, maybe this?:

```markdown
The destructor for a wrapper object will run when the object is
garbage-collected.
```

Leave it as an exercise for the reader to figure out how to force garbage collection for testing purposes. People can find that in blog posts and StackOverflow etc. (For example, https://stackoverflow.com/q/27321997/436641.) But we don't have to document it.

Reply:
Agree with leaving reader to figure it out since we are not recommending to use command line flags outside of testing. First options is informative and I'll proceed with that.